lwr_hw/src/lwr_hw.cpp

```diff
@@ -287,6 +289,18 @@ namespace lwr_hw
       position_cart_interface_.registerHandle(cart_wrench_handle);
     }
 
+    for (int i=0; i< n_joints_; i++)
+      for (int j=0; j< n_joints_; j++)
+	{
+	hardware_interface::JointHandle inertia_handle_ij;
+	inertia_handle_ij = hardware_interface::JointHandle(hardware_interface::JointStateHandle(
```

Comment:
This is somehow not natural IMHO, a joint handle is meant to be used for a joint.. I know I have done things like this before, but we should try to avoid those workarounds

Why don't we add a `#include <realtime_tools/realtime_publisher.h>`  to publish `robot_info` topic (similar to the `camera_info` in camera drivers) or any other solution?

Also, I think the efforts should be in improving the dynamic model URDF-wise, and more and more forget about the FRI side, thus these inertia values could be computed from there, what do you think?